wordpress-mobile/WordPress-iOS
# Jetpack Section: Scan: Groups threats found by date

Project #15190
---

Depends on: https://github.com/wordpress-mobile/WordPress-iOS/pull/15755

Based on some design feedback this groups the threats found by date using the same logic as the threat history vew. 

### Screenshots

| Idle State | Fixing State |
|:---|:---|
|![Simulator Screen Shot - iPhone 11 Pro - 2021-02-02 at 21 30 20](https://user-images.githubusercontent.com/793774/106689366-e28b8f80-659d-11eb-8abe-3115bc39da98.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-02-02 at 21 29 03](https://user-images.githubusercontent.com/793774/106689401-f2a36f00-659d-11eb-9ab6-61f2c40420ae.png)|



### To test:
1. Launch the app
2. Tap on My Site
3. Select a site with Jetpack Scan enabled
4. Tap on Scan
5. 👁️ The threats are now grouped by date
6. Tap History
7. Verify the threats are still correctly grouped
8. Tap back
9. Tap Fix all
10. Verify the threats being fixed are no longer groups in a section

### PR submission checklist:
